public/keymaps/h/handwired_swiftrax_cowfish_default.json

```diff
@@ -1,19 +1,19 @@
 {
```

Comment:
This keymap won't work unless and until https://github.com/qmk/qmk_firmware/pull/18984 is back-ported to `qmk:master`.